routes/quizzes.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+use App\Http\Controllers\QuizController;
+
+Route::multilingual('/quizzes/{quiz}', [QuizController::class, 'show'])
```

Comment:
This should probably be under `courses/{course}/quiz` and possibly one similar to this but for modules. It seems there can only be one quiz so you probably don't need to pass in the ID, but if you need to you could do that too I guess.